src/index.html

```diff
@@ -3,11 +3,311 @@
+                  alt="bike handle bars"
+                  class="details__image"
+                >
+              </div>
+            </div>
+            <div class="details__content">
+              <h3 class="details__content-heading">Auto Unlock</h3>
+
+              <p class="details__content-text">The app senses when you're nearby to unlock automatically. GPS tracking so
+                you know where your bike is and can track it anytime.
+              </p>
+            </div>
+          </section>
+
+          <section>
+            <div class="details__row">
+              <div class="details__photo details__photo--cols--1-4">
+                <img
+                  src="./images/photos/3.jpg"
+                  alt="bike from the back"
+                  class="details__image details__image--big">
+              </div>
+              <div class="details__photo details__photo--cols--5-6">
+                <img
+                  src="./images/photos/4.jpg"
+                  alt="close up of front of bike"
+                  class="details__image">
+              </div>
+            </div>
+            <div class="details__content">
+              <h3 class="details__content-heading">Range & Integrated lights</h3>
+
+              <p class="details__content-text">The removable battery has up to 70km battery autonomy and weighs only 2.4
+              kg. Lights integrated into the frame give you always-on visibility day and night.
+              </p>
+            </div>
+          </section>
+
+
+          <section>
+            <div class="details__row">
+              <div class="details__photo details__photo--cols--1-2">
+                <img
+                  src="./images/photos/5.jpg"
+                  alt="close up of bike brake"
+                  class="details__image">
+              </div>
+              <div class="details__photo details__photo--cols--3-6">
+                <img
+                  src="./images/photos/6.jpg"
+                  alt="close up of bike stand"
+                  class="details__image details__image--big">
+              </div>
+            </div>
+            <div class="details__content">
+              <h3 class="details__content-heading">Hydraulic disc brakes & Lightweight</h3>
+
+              <p class="details__content-text">Brakes with total stopping power the second you make contact. The removable
+                battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you
+                always-on visibility day and night.
+              </p>
+            </div>
+          </section>
+
+          <a href="#" class="button details__button">Explore</a>
+        </div>
+      </section>
+
+      <section class="contact-us" id="contact-us">
+        <div class="contact-us__wrapper">
+          <h2 class="title contact-us__title">Contact us</h2>
+
+          <div class="contact-us__content-wrapper">
+            <form
```

Comment:
checklist suggests
>15 Page shouldn't be reloaded on form submit:

currently it does reload the page. You could try js or a more simple inline html approach:
`<form onsubmit="document.querySelector('<form_selector>').reset()"`
so that form validate inputs, and responds to submit action, yet not reloading the page